Remotion/Web/Core/ExecutionEngine/Infrastructure/WxeExecutor.cs

```diff
@@ -119,8 +119,10 @@ public void ExecuteFunctionExternal (WxeFunction function, Control sender, WxeCa
           href,
           options.Target,
           options.Features ?? string.Empty);
-      _page.ClientScript.RegisterClientScriptBlock (_page, typeof (WxeExecutor), "WxeExecuteFunction", openScript);
-      _page.RegisterClientSidePageEventHandler (SmartPageEvents.OnLoaded, "WxeExecuteFunction", functionName);
+
+      var key = Guid.NewGuid().ToString();
```

Comment:
The code works as intended, I can't think of any bad side effects. As an improvement tho, a new guid is created a few lines above, you could store its string in a variable and reuse it here.